WordPress/Classes/ViewRelated/Site Creation/Verticals/VerticalsCell.swift

```diff
@@ -2,23 +2,57 @@ import UIKit
 
 class VerticalsCell: UITableViewCell, ModelSettableCell {
     @IBOutlet weak var title: UILabel!
+    @IBOutlet weak var subtitle: UILabel!
+
+    private struct Constants {
+        static let verticalSubtitle = ""
+        static let newVerticalSubtitle = NSLocalizedString("Custom category", comment: "Placeholder for new site types when creating a new site")
+    }
 
     var model: SiteVertical? {
         didSet {
             title.text = model?.title
+            if let isNew = model?.isNew, isNew == true {
```

Comment:
While we anticipate only having 5 rows in our table of Verticals, scroll performance will be impacted by the resizing going on here. 

What if we created two cell types: one for known verticals and one for the alternate type (i.e., "custom category")? I think this would help ensure that the row heights are correct. They look a bit different than the visual spec at the moment.